src/docbuilder/rustwide_builder.rs

```diff
@@ -53,8 +53,13 @@ impl RustwideBuilder {
 
         let mut builder = WorkspaceBuilder::new(&config.rustwide_workspace, USER_AGENT)
             .running_inside_docker(config.inside_docker);
-        if let Some(custom_image) = &config.local_docker_image {
-            builder = builder.sandbox_image(SandboxImage::local(&custom_image)?);
+        if let Some(custom_image) = &config.docker_image {
+            let image = match SandboxImage::local(&custom_image) {
+                Ok(i) => i,
+                Err(CommandError::SandboxImageMissing(_)) => SandboxImage::remote(custom_image)?,
+                Err(err) => return Err(err.into()),
+            };
```

Comment:
I don't think this is what we want: this would result in the image never being updated, as once the image is downloaded it's available locally. A good heuristic could be to treat an image as remote if it contains `/`.

Reply:
> A good heuristic could be to treat an image as remote if it contains /.

Hmm, there are plenty of remote images that don't have `/` though, like `ubuntu` or `alpine`. Personally I would be ok with it being cached indefinitely; you can always remove the image if it gets out of date. Is there a way to tell if a local image was built locally or downloaded?

Reply:
The full name for `ubuntu` is `docker.io/ubuntu`, docker just has a hardcoded "try `docker.io` if no registry specified". That said, just using a locally cached image without checking that it's up to date is pretty standard behaviour of docker tooling, you either need to explicitly `docker pull ...` to update it, or pass `--pull` to commands like `docker build` to have them attempt pulling updates.

Reply:
> The full name for ubuntu is docker.io/ubuntu, docker just has a hardcoded "try docker.io if no registry specified".

Well yes, but if someone overrides the image, they'll be using the shorthand.

Reply:
Honestly I'm not sure: I don't want us accidentally use an older image in production because we end up changing the environment variable without realizing it.

> Hmm, there are plenty of remote images that don't have / though, like ubuntu or alpine.

I don't think that's a problem, you can't realistically build any Rust crate using those images.

Reply:
> Honestly I'm not sure: I don't want us accidentally use an older image in production because we end up changing the environment variable without realizing it.

Oh I see - I don't think this will have any impact on prod, we don't override the image there.

Reply:
If we *do* want to change the image in prod, I think we should do that by changing the default, not through environment variables.

Reply:
Hmm, ok.